src/NHSD.BuyingCatalogue.Solutions.Persistence/Repositories/PriceRepository.cs

```diff
@@ -11,26 +11,45 @@ public sealed class PriceRepository : IPriceRepository
     {
         private readonly IDbConnector _dbConnector;
 
-        private const string ListCataloguePricesSql = @"
-        SELECT  cp.CatalogueItemId,
-                ci.[Name] AS CatalogueItemName,
-                cp.CataloguePriceId,
-                cp.CataloguePriceTypeId,
-                pu.PricingUnitId,
-                pu.[Name] AS PricingUnitName,
-                pu.[Description] AS PricingUnitDescription,
-                pu.TierName AS PricingUnitTierName,
-                cp.TimeUnitId,
-                cp.CurrencyCode,
-                cp.Price AS FlatPrice,
-                cptr.BandStart,
-                cptr.BandEnd,
-                cptr.Price AS TieredPrice
-        FROM    dbo.CataloguePrice AS cp
-                INNER JOIN dbo.PricingUnit AS pu ON pu.PricingUnitId = cp.PricingUnitId
-                INNER JOIN dbo.CatalogueItem AS ci ON ci.CatalogueItemId = cp.CatalogueItemId
-                LEFT OUTER JOIN dbo.CataloguePriceTier AS cptr ON cptr.CataloguePriceId = cp.CataloguePriceId
-        WHERE   cp.CatalogueItemId = @solutionId;";
+        private const string ListCataloguePricesSql = @"SELECT cp.CatalogueItemId,
```

Comment:
Can you put the formatting back to how it was, as this is the formatting structure for the repository